site/docs/reference/Connectors/capture-connectors/amazon-s3.md

```diff
@@ -138,14 +133,43 @@ Options are:
    For now, use a prefix in the endpoint configuration to limit the scope of each capture to data of a single file type.
    :::
 
-Only CSV data requires further configuration. When capturing CSV data, you must specify:
+#### CSV configuration
+
+Only CSV data allows further configuration. When capturing CSV data, you may additionally specify:
+
+* **Delimiter**. Options are:
+  * Comma (`","`)
+  * Pipe (`"|"`)
+  * Space (`"0x20"`)
+  * Semicolon (`";"`)
+  * Tab (`"0x09"`)
+  * Vertical tab (`"0x0B"`)
+  * Unit separator (`"0x1F"`)
+  * SOH (`"0x01"`)
+  * Auto
 
-* **Delimiter**
 * **Encoding** type, specified by its [WHATWG label](https://encoding.spec.whatwg.org/#names-and-labels).
+
 * Optionally, an **Error threshold**, as an acceptable percentage of errors.
```

Comment:
I kinda feel like Error Threshold could use more explanation, since it's pretty weird. It allows you to skip CSV rows that fail to parse, which is a pretty niche use case. It could probably do with some sort of warning, too, since by setting this to a value other than `0`, you're opting in to throwing away data in malformed rows (instead of stopping so you can fix them).